mlflow/server/js/src/components/NoExperimentView.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import Colors from '../styles/Colors';
+import overflow from '../static/project.svg';
+
+export default class NoExperimentView extends Component {
+  render() {
+    return (
+      <div>
+        <img
+          className='center'
+          alt="404 Not Found"
```

Comment:
It looks like `alt` is also used for accessibility purposes (i.e. screen readers will read out the text in the alt tag): https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#Attributes.

Maybe instead we could put a description here like "No experiments exist"? Also, the icon for some reason doesn't appear as it does in your screenshot for me (i.e. the colors & stuff are missing). Not a big deal but just wanted to make sure you knew:

![screen shot 2018-10-09 at 2 26 55 pm](https://user-images.githubusercontent.com/2358483/46699697-6b14e380-cbcf-11e8-947f-4d6a8ae220dd.png)

Reply:
Forgot to add the new image! It's there now.